src/state/nftMarket/hooks.ts

```diff
@@ -0,0 +1,10 @@
+import { useEffect } from 'react'
+import { useAppDispatch } from 'state'
+import { fetchCollections } from './reducer'
+
+export const useFetchCollections = () => {
+  const dispatch = useAppDispatch()
+  useEffect(() => {
+    dispatch(fetchCollections())
```

Comment:
doesn't `fetchCollections()` already dispatches the action inside?

Reply:
it's an action as well (a thunk) so it needs to be dispatched